packages/nx-plugin/executors.json

```diff
@@ -0,0 +1,9 @@
+{
+  "executors": {
+    "autorun": {
+      "implementation": "./src/executors/autorun/executor",
+      "schema": "./src/executors/autorun/schema.json",
+      "description": "autorun executor"
```

Comment:
Could you add a more useful description? I already know I'm looking at an executor called autorun, so this description tells me nothing.

Reply:
Added better text